pkg/test/framework/components/istio/config.go

```diff
@@ -106,6 +132,12 @@ type Config struct {
 	// Indicates that the test should deploy Istio into the target Kubernetes cluster before running tests.
 	DeployIstio bool
 
+	// Indicates that the test should deploy Istio using the alpha installer.
+	AlphaInstaller bool
```

Comment:
Assuming the new installer is a "done deal", I'd reverse this and say "OldInstaller" to refer to the current installer. Once it is gone, then simply removing the flags will be enough to keep the code clean.

Reply:
Sounds good. One question I have about this - there is a decent chunk of code needed for the old installer. Do you think all of this should be changed to be `old-...` instead of the new stuff being `new-...`?